src/components/tags-input/TagsInput.js

```diff
@@ -91,6 +90,10 @@ export default class TagsInput extends PureComponent {
     this.props.deleteTag(item)
   }
 
+  clearFilter() {
```

Comment:
What's the reason for extracting it to a separate function?

Reply:
The problem was with usage of redundant `shouldComponentUpdate` (no redundant usage component update). Firstly I fixed it with replacing `PureComponent` with `Component`, but then new error appeared - https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-will-update-set-state.md. And I decided to save `PureComponent` and extracted setting new state to new function :D